_posts/2024-02-12-introducing-kimai-mobile-time-tracking-features.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Introducing Kimai Mobile Time Tracking Features "
+date: "2024-02-12 12:00:00 +0200"
+author: cloudrizon
+tags: [Apps & Plugins]
+---
+
+## Introducing Kimai Mobile Time Tracking Features 
+
+Kimai Mobile released a couple of new app features that our team is excited to share with you today. 
+
+Kimai Mobile seamlessly integrates with the Kimai Time Tracker Server, providing users with the convenience of managing multiple workspaces concurrently. This allows for efficient time logging across various projects and clients. All time tracking data is securely stored on the self-hosted Kimai Server, ensuring both data privacy and accessibility. 
```

Comment:
```suggestion
is securely stored on the self-hosted Kimai Server 
```
Also when using the SaaS version

Reply:
Can you clarify this point more? What should be modified?

Reply:
Ups, something went wrong with the suggestion. Updated it.